packages/formspree-react/package.json

```diff
@@ -66,9 +67,6 @@
     "tsup": "^6.2.2",
     "typescript": "^4.7.4"
   },
-  "peerDependencies": {
-    "@stripe/stripe-js": "^1.29.0"
-  },
```

Comment:
Just to be sure, we need stripe-js as a direct dependency, peerDependency doesn't work, right?